### PR TITLE
Stopping AWS Janitor from removing default IGW

### PR DESCRIPTION
**Pull request by chrislovecnm**
This PR fixes the AWS Janitor so that it does not remove the Internet
Gateway for the default VPC.

Fixes https://github.com/kubernetes/test-infra/issues/6573

### CODE REVIEW

**Comment by justinsb** [comment]
Is there an issue to document why we are doing this?

**Comment by chrislovecnm** [comment]
@justinsb added the issue.  The IGW's on the default VPCs are being deleted by the janitor.  I noticed this when I launch an instance to test kubetest.

File: maintenance/aws-janitor/main.go [hunk 1]
```diff
@@ -466,6 +486,10 @@ func (internetGateways) MarkAndSweep(sess *session.Session, acct string, region
 					glog.Warningf("%v: detach from %v failed: %v", i.ARN(), *att.VpcId, err)
 				}
 			}
+			if isDefault {
+				glog.Infof("%s: skipping delete %T: %v", i.ARN(), ig, ig)
```
Comment (oomichi) [inline maintenance/aws-janitor/main.go]: I think we will see the same duplicated message if isDefault is true, because of line 478.
Is that intentional?

Comment (chrislovecnm) [reply maintenance/aws-janitor/main.go]: More context please

Comment (chrislovecnm) [reply maintenance/aws-janitor/main.go]: I dropped one of the messages

**Comment by krzyzacy** [comment]
`:0.4` image pushed

**Pull request merged**